### PR TITLE
landing page: highlight verified community in communities list

### PR DESCRIPTION
**Pull request by anikachurilova**
* closes https://github.com/zenodo/zenodo-rdm/issues/717

Screenshot 2024-02-19 at 11 27 41

### CODE REVIEW

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/CommunitiesManagement.js [hunk 1]
```diff
@@ -126,6 +126,7 @@ export class CommunitiesManagement extends Component {
               error={error}
               loading={loading}
               maxDisplayedCommunities={MAX_COMMUNITIES}
+              branded={record.parent?.communities?.default}
```
Comment (alejandromumo) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/CommunitiesManagement.js:129]: question 1: only one community can have the `verified` icon?

question 2: why is the `default` community the one considered `branded`?

**Comment by alejandromumo** [comment]
Reviewed with @yashlamba

**Review by 0einstein0 — APPROVED** [review]
LGTM, reviewed with @jrcastro2

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/CommunitiesManagement.js (hunk 1 above)
Comment (alejandromumo) [reply invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/CommunitiesManagement.js:129]: About (1) - just as info, records already have a `verified` field but it's related to content moderation.

**Pull request merged**